backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/EligibilityController.java

```diff
@@ -94,33 +90,33 @@
 	 * @return The result of the query
 	 */
 	@PostMapping("/inquire-phn")
-	public ResponseEntity<InquirePhnResponse> inquirePhn(@Valid @RequestBody InquirePhnRequest inquirePhnRequest) {
+	public ResponseEntity<InquirePhnResponse> inquirePhn(@Valid @RequestBody InquirePhnRequest inquirePhnRequest, HttpServletRequest request) {
+		Transaction transaction = transactionStart(request, TransactionType.PHN_INQUIRY);
+		inquirePhnRequest.getPhns().forEach(phn -> {
```

Comment:
lambda can be simplified 
`inquirePhnRequest.getPhns().forEach(phn -> addAffectedParty(transaction, IdentifierType.PHN, phn));`